packages/@aws-cdk/cfnspec/spec-source/specification/000_cfn/500_WAFv2_RuleGroup_Rename_Properties_patch.json

```diff
@@ -0,0 +1,102 @@
+                "op": "remove",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.CaptchaAction"
+            },
+            {
+                "op": "add",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.Captcha",
+                "value": {
+                    "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-captchaaction.html",
+                    "Properties": {
+                        "CustomRequestHandling": {
+                            "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-captchaaction.html#cfn-wafv2-rulegroup-captchaaction-customrequesthandling",
+                            "Required": false,
+                            "Type": "CustomRequestHandling",
+                            "UpdateType": "Mutable"
+                        }
+                    }
+                }
+            },
+            {
+                "op": "remove",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.ChallengeAction"
+            },
+            {
+                "op": "add",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.Challenge",
+                "value": {
+                    "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-challengeaction.html",
+                    "Properties": {
+                        "CustomRequestHandling": {
+                            "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-challengeaction.html#cfn-wafv2-rulegroup-challengeaction-customrequesthandling",
+                            "Required": false,
+                            "Type": "CustomRequestHandling",
+                            "UpdateType": "Mutable"
+                        }
+                    }
+                }
+            },
+            {
+                "op": "remove",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.CountAction"
+            },
+            {
+                "op": "add",
+                "path": "/PropertyTypes/AWS::WAFv2::RuleGroup.Count",
+                "value": {
+                    "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-countaction.html",
+                    "Properties": {
+                        "CustomRequestHandling": {
+                            "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-rulegroup-countaction.html#cfn-wafv2-rulegroup-countaction-customrequesthandling",
+                            "Required": false,
+                            "Type": "CustomRequestHandling",
+                            "UpdateType": "Mutable"
+                        }
+                    }
+                }
+            }
+        ]
+    }
+}
```

Comment:
We do not need to fix [types](https://github.com/aws/aws-cdk/commit/affe040c8443be074822254d1e75a28b264cd801#diff-827a2fd012e049c7ccedffa0360c12e7d967a173f36b8150de73ef6adc42ee4cL885-L915) with this patch since those already get replaced in [another patch](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cfnspec/spec-source/specification/000_cfn/500_Revert_To_Json_Types_patch.json#L876-L918)